.github/workflows/python-app.yml

```diff
@@ -0,0 +1,106 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  linux:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Add release key
+      run: echo ${{ secrets.RELEASE_KEY }} > keys/build_key.txt
+    
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements_full.txt
+        
+    - name: Build Linux App
+      run: python -m PyInstaller wwrando.spec
+    - name: Bundle Linux App
+      run: python build.py
+    - uses: actions/upload-artifact@v2
+      with:
+        name: wwrando-linux
+        path: dist/*.zip
+
+  macos:
+
+    runs-on: macos-latest
```

Comment:
It would be nice to be more consistent with the line breaks throughout this file.

Reply:
Should be better now